### PR TITLE
Add friendly error messages for incorrect options values

### LINKED ISSUE #1483: More user-friendly message on invalid CLI arguments
Strange or invalid error on CLI argument:
- [ ] `--custom-formatter` (need more user-friendly, like do `eslint` - `package not found, maybe you should run npm install stylelint-formatter-test`)
- [ ] `--formatter` (invalid error)
```shell
stylelint '**/*.css' --formatter test
```
```shell
TypeError: chosenFormatter is not a function
    at /home/itgalaxy/web/dev0.cms.dev.itgalaxy.company/public_html/wp-content/themes/insteria/node_modules/stylelint/dist/standalone.js:58:20
```
- [ ] `--ignore-path` (no errors if file not found in path).
```shell
stylelint '**/*.css' --ignore-path fakepath
```
- [ ] `--syntax` (no throw error if syntax incorrectly)
```shell
stylelint '**/*.css' --syntax fake
```

### PR DESCRIPTION
**Pull request by jeddy3**
Closes #1483

### CODE REVIEW

**Comment by evilebottnawi** [comment]
LGTM

File: src/standalone.js [hunk 1]
```diff
@@ -54,9 +54,17 @@ export default function ({
       })
     }
 
-    const chosenFormatter = (typeof formatter === "string")
-      ? formatters[formatter]
-      : formatter
+    let chosenFormatter = ""
```
Comment (davidtheclark) [inline src/standalone.js]: This is kind of a confusing default value. I'd say leave it undefined (`let chosenFormatter`) or set `let chosenFormatter = formatter` and just change it if it's a string.

**Comment by davidtheclark** [comment]
I had the one comment. Feel free to merge after addressing it.

File: src/standalone.js (hunk 1 above)
Comment (jeddy3) [inline src/standalone.js]: Much nicer. Updating now.

**Pull request merged**